phabricator.py

```diff
@@ -37,8 +39,25 @@ def run(self):
         # Find the preselected branch
         git_branch = settings.get('branch')
 
-        # If no preselected branch is provided
-        if git_branch == None:
+        if git_branch is None and settings.get('use_arc_land_onto_default', False):
```

Comment:
It is a pattern I picked up from `jscs`. The overloading is fine here since `branch` as a string and as a dict are mutually exclusive.

http://jscs.info/rules.html#disallowspacesinsidebrackets

That being said, I understand where you are coming from. I was trying to keep the options somehow related. Thoughts on renaming the options to `branch_use_arc_land_onto_default`?